stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/ReactionClickListener.kt

```diff
@@ -0,0 +1,7 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import io.getstream.chat.android.client.models.Reaction
+
+public fun interface ReactionClickListener {
+    public fun onReactionClick(reaction: Reaction)
+}
```

Comment:
Was the intention here to reuse this in multiple places? `ReactionsView` still has its own copy of this interface nested inside it.

Reply:
Yep, making careless mistakes. Removed redundant `ReactionClickListener` in `ReactionsView `